### ISSUE TITLE
Hangs scrolling settings

### ISSUE THREAD

**Issue by msp42** [ifrotz/iosfrotz#308]
Frotz 1.8.5
iOS 13.6.1
iPhone 7 Plus

Every time I go to the Settings menu and scroll below 
Story Info Editing it stops responding.

**Comment by msp42** [comment]
Apart from the hang, Frotz is working very well. Great work!

**Comment by spathiwa** [comment]
So, I don't have a physical iPhone 7 Plus or other large-screen iPhone, but I was able to reproduce a hang in the emulator, but only in Landscape mode.  If I stick to portrait, the hang doesn't seem to happen.

Does that match what you're seeing? 

The hang I've reproduced seems to be stuck in a iOS routine which automatically lays out views; there's no actual Frotz code on the stack, so it's probably an iOS bug, but I still may be able to figure out how to work around it and patch for it.

**Comment by spathiwa** [comment]
Scratch that.  Under iOS 10.x (which shipped with the iPhone 7 Plus and is what the emulator was using by default) it happens only on landscape, but under iOS 13.x it hangs in portrait mode too.

Fortunately, I figured out the issue and have a fix.  Turns out it was Frotz's fault after all, although arguably the OS still shouldn't hang because of it.

Gonna wait a couple days and see if any other serious issues worth fixing show up, and then put up a new version.

Thanks for reporting this!

**Comment by msp42** [comment]
Thank you for all your hard work!
Frotz is ace!!

Cheers
Mark Phillips

Sent from my iPhone

On 5 Sep 2020, at 08:44, Craig Smith <notifications@github.com> wrote:

﻿

Scratch that. Under iOS 10.x (which shipped with the iPhone 7 Plus and is what the emulator was using by default) it happens only on landscape, but under iOS 13.x it hangs in portrait mode too.

Fortunately, I figured out the issue and have a fix. Turns out it was Frotz's fault after all, although arguably the OS still shouldn't hang because of it.

Gonna wait a couple days and see if any other serious issues worth fixing show up, and then put up a new version.

Thanks for reporting this!
